code/src/redux/reducers/bag.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const bag = createSlice({
+  name: 'bag',
+  initialState: {
+    items: [
+      //{ id: 1, name: 'Handduk', inBag: false },
+      //{ id: 2, name: 'Tandborste', inBag: false },
+      //{ id: 3, name: 'Tvål', inBag: false },
+    ]
+  },
+  reducers: {
+    addThings: (state, action) => {
```

Comment:
Minor detail, but this adds just one thing at a time so it should perhaps be named `addThing`